docs/src/components/landing-page/TwitterShowcaseCards.vue

```diff
@@ -36,65 +37,82 @@
     </template>
     <q-carousel-slide :name="slideIndex" class="showcase-cards text-size-10" v-for="(tweetGroup, slideIndex) in tweetGroups" :key="`slide-${slideIndex}`">
       <div class="carousel-grid">
-        <div v-for="(tweetId, cardIndex) in tweetGroup" :key="`twitter-card-${cardIndex}`">
-          <twitter-card :tweetId="tweetId"/>
+        <div v-for="(tweetId, cardIndex) in tweetGroup" :key="`twitter-card-${cardIndex}`" class="tweeter-tweet">
+          <div :id="`tweet-container-${tweetId}`"></div>
         </div>
       </div>
     </q-carousel-slide>
   </q-carousel>
 </template>
 
-<script>
-import { defineComponent, ref } from 'vue'
-import TwitterCard from 'src/components/landing-page/TwitterCard'
+<script lang="ts">
+import { defineComponent, onMounted, ref } from 'vue'
 
-// TODO: fix the issue causing ONLY the first tweet group (slide 1) display
 const scriptElement = document.createElement('script')
 scriptElement.setAttribute('src', 'https://platform.twitter.com/widgets.js')
 scriptElement.setAttribute('charset', 'utf-8')
 document.head.appendChild(scriptElement)
 
+const addTwttr = async () => {
```

Comment:
I'd use `getTwitterInstance` or similar

Reply:
You should use `function` here too, or remove the code block and just return the promise
Using `function` is preferred tho